cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/analytics/BulkWriteUdtTest.java

```diff
@@ -59,6 +60,61 @@ class BulkWriteUdtTest extends SharedClusterSparkIntegrationTestBase
                                                      + "           id BIGINT PRIMARY KEY,\n"
                                                      + "           nested " + NESTED_FIELD_UDT_NAME + ");";
 
+    // UDT with list, set and map in it
+    public static final String UDT_WITH_COLLECTIONS_TYPE_NAME = "udt_with_collections";
+    public static final String UDT_WITH_COLLECTIONS_TYPE_CREATE = "CREATE TYPE " + TEST_KEYSPACE + "." + UDT_WITH_COLLECTIONS_TYPE_NAME +
+            " (f1 list<text>, f2 set<int>, f3 map<int, text>);";
+
```

Comment:
It would be nice to use QT to fuzz test the different combinations of types. Just say it out load, no need to address.